R/loocv.R

```diff
@@ -42,8 +42,8 @@
 #' @author Witold Wiecek
 #' @importFrom utils txtProgressBar
 #' @importFrom utils setTxtProgressBar
+#' @references Gelman, Andrew, et al. Bayesian data analysis. Chapman and Hall/CRC, 2013.
```

Comment:
Rather than citing BDA, "expected log predictive density (ELPD) for that group (see Gelman et al 2013)" should be a reference to 

Gelman, Andrew, Jessica Hwang, and Aki Vehtari. “Understanding Predictive Information Criteria for Bayesian Models.” Statistics and Computing 24, no. 6 (November 2014): 997–1016. https://doi.org/10.1007/s11222-013-9416-2.

It's already hyperlinked in the help doc, we could harmonise that!

Reply:
Ah ok, I wasn't sure what reference you wanted.

Reply:
I'll pull down the new devel just to make sure.